controller.py

```diff
@@ -80,3 +81,23 @@ def menu_info(): -> list
             menu_info_list.append(menu_info_str)
 
         return menu_info_list
+
+    def more_info(): -> list
```

Comment:
The colon needs to be after the return object.
def more_info() -> list:

Reply:
oh oops. Im not sure how I missed that

Reply:
@simonaiqbal please take a look at my View class to write some functions in Controller